### PR TITLE
upgrade golangci-lint to 1.52

### PR DESCRIPTION
**Pull request by zeitlinger**
Based on #126

### CODE REVIEW

**Review by rgeyer — APPROVED** [review]

File: lint/rule_target_job_instance.go
Comment (rgeyer) [inline lint/rule_target_job_instance.go:15]: I presume that the TODOs were removed as a result of the updated linting? Can we override that rule?

All of the TODOs which were removed are still not complete, and.. I've been a poor maintainer in that they aren't tracked as issues either 🤦

**Comment by rgeyer** [comment]
One question on the TODOs. I approved pre-maturely assuming that this only upgraded the linter, without necessarily having any code changes.

File: lint/rule_target_job_instance.go
Comment (zeitlinger) [reply lint/rule_target_job_instance.go:15]: yes, we can override that rule - I'll do that

**Pull request merged**
